packages/core/test/types.test.ts

```diff
@@ -253,23 +253,23 @@ describe('Raise events', () => {
               {
                 actions: raise({
                   type: 'ALOHA'
-                }) as any /* TODO: FIX */,
+                }),
```

Comment:
I created a spike for type-related improvements around this here: https://github.com/statelyai/xstate/pull/3694